packages/deployer/subtasks/prepare-deployment.js

```diff
@@ -28,7 +28,7 @@ subtask(
 ).setAction(async (taskArguments, hre) => {
   const { clear, alias } = taskArguments;
 
-  const deploymentsFolder = hre.deployer.paths.deployments;
+  const deploymentsFolder = hre.config.deployer.paths.deployments;
```

Comment:
this is going to break because this variable doesn't include the network and instance subpaths

Reply:
Fixed